wpeverest/everest-forms
# Dev - CleverReach Integration

### All Submissions:


### Changes proposed in this Pull Request:

This PR introduces the new cleverreach integration.
Create account : - https://eu2.cleverreach.com/admin/
![image](https://github.com/user-attachments/assets/e60906c4-cc9b-4dff-ab28-429a6bfcec85)
![image](https://github.com/user-attachments/assets/b77af275-66b9-4c3d-a719-347507b15ba1)

### Depedency PR:
https://github.com/wpeverest/everest-forms-pro/pull/962
 
### How to test the changes in this Pull Request:

1. Goto Create account > login > dashboard > Recipients > Add one list.
2. After adding list goto > my account > interfaces > RestApi > Create auth ( not need to put redirect url leave as it is) > choose scopes > create
3. Goto Settings > Integration > CleverReach > connect  
4.Go to the form setting > integration > clevereach > choose field to sync and for custom field create attributes from check screenshots
5. After submission check the list.

### Types of changes:



### Other information:


### Changelog entry

> Dev  - CleverReach integration.
